### ISSUE TITLE
js: refactor all constants into a single file

### ISSUE THREAD

**Issue by algomaster99** [iterative/dvc.org#714]
It is nice to have a single source for all constants like `ROOT_ELEMENT`, etc. to make it feasible for resuing at different places.

**Comment by shcheklein** [comment]
not sure this a good first issue. @algomaster99 do you have a list of constants we want to move?

**Comment by algomaster99** [comment]
- `ROOT_ELEMENT`
- `SIDEBAR_MENU`
- `MIN_HEIGHT`

And there are a few more. These are a few examples.

**Comment by jorgeorpinel** [comment]
Nice. If you can list them all in as checkboxes (and check off the ones you fix with PRs), then it will be a good first issue 🙂

**Comment by shcheklein** [comment]
there also should be a list of where to _use_ these constants, right?

**Comment by algomaster99** [comment]
@shcheklein We can determine it by grepping through the whole project for the particular constant.
Although, there are some constants which are used only once in a project. So I will have to decide whether they should really go into the file or not.

**Comment by shcheklein** [comment]
@algomaster99 sometimes even grep can be not enough if it's already a value that we calculated using a few constants (like in your tooltip code as far as I understand).

**Comment by algomaster99** [comment]
@shcheklein in my case, we are just movie the header element' class to a constant file. Not the height because we can only find its height after it has mounted. It is similar to what we have done for `bodybag`.

> sometimes even grep can be not enough

Might be. I will have to look carefully and jot them down here. :sweat_smile:

**Comment by jorgeorpinel** [comment]
> there are some constants which are used only once in a project

My 2c on this is that either the constant goes in the shared file like all the others (in case it's reused later), or the value is hardcoded as a literal where it's used if you're sure no one else will ever need it.

**Comment by jorgeorpinel** [comment]
Where should the constants file be located? Is there a best practice for React or Next project structures?

See https://github.com/iterative/dvc.org/pull/717/files#diff-7c51329997e372ad3b55f13cecdd28ef

**Comment by shcheklein** [comment]
let's revisit this, might be not relevant with the latest migration
